v3python/codegen/affine.py

```diff
@@ -83,7 +83,8 @@ def write_shim_source(self, functionals, fout):
         shared_iface_family = akdesc.SHARED_IFACE.FAMILY if shared_iface else akdesc.FAMILY
         validator_defs, capability_table_entries = self.codegen_capability_table(functionals)
         meta_cos = self.codegen_compact_kernels()
-        packed_string = self._this_repo.get_data('affine_kernel_packed_string')
+        packed_string = self._this_repo.get_data('affine_kernel_packed_string', return_none=True)
+        packed_string = '""' if packed_string is None else packed_string
```

Comment:
Do not add commit `40bb2152e236a51477a772b54cc0fdcd12c79823` and `abb62c8cf18cedc1ca7457b05dacbe87653d626b` to this PR.

Reply:
It won't build on Windows without those two commits. Perhaps we can wait until those two commits are merged from your branch?